sdgym/benchmark.py

```diff
@@ -538,14 +538,14 @@ def _create_instance_on_ec2(script_content):
     echo "{script_content}" > ~/sdgym_script.py
     echo "======== Run Script ==========="
     sudo python3 ~/sdgym_script.py
-    echo "======== Terminate ==========="
+    echo "======== Complete ==========="
     INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
     aws ec2 terminate-instances --instance-ids $INSTANCE_ID
     """
 
     response = ec2_client.run_instances(
         ImageId='ami-07d9b9ddc6cd8dd30',
-        InstanceType='t2.medium',
+        InstanceType='t2.xlarge',
```

Comment:
@lajohn4747 I think this instance type would not work. From what I can gather this is an instance that does not have GPUs and is from this table shown here: https://aws.amazon.com/fr/ec2/instance-types/t2/  if that is the case, it may not work still. 

I think the goal is to pick larger instance with GPUs from here: https://aws.amazon.com/fr/ec2/instance-types/p2/
I think in the past we used ``p2.xlarge`` . I'd ask @npatki about this.  

One another question I have is we are using an ``ami``. Would it be possible to do a fresh install of SDGym within the launched instance. An AMI would need to be maintained.

Reply:
Sounds good! I agree it might take too long based on Felipe's findings. Will update and spin up a run with `p2.xlarge` to take advantage of the larger processor and the GPU.

The `ami` is just an ID to specify the default image for the virtual machine. This command does install a fresh SDGym from the mainline onto the machine so we aren't maintaining anything outside of SDgym